client/src/ui/_vars.scss

```diff
@@ -159,6 +159,7 @@ $mdn-theme-light-text-active: #{$mdn-color-neutral-50};
 $mdn-theme-light-text-inactive: #{$mdn-color-neutral-40}a6;
 $mdn-theme-light-text-link: $mdn-color-light-theme-blue-60;
 $mdn-theme-light-text-invert: $mdn-color-white;
+$mdn-theme-light-text-muted: #6f6f6f;
```

Comment:
Nit: I wouldn't add further sass variables if we don't need them anywhere, just define the css variable directly

Reply:
I'll keep it, as I had wanted to use this color in other circumstances before (being familiar with Bootstrap).